app/assets/stylesheets/application.scss

```diff
@@ -19,10 +19,11 @@ h1, h2{
     color: $dark-yellow-text;
 }
 
-h2.dark-background-text{
+h1.dark-background-text, h2.dark-background-text, p.dark-background-text{
```

Comment:
seems like you might be able to define `.dark-background-text { color: $light-text; }` without needed to target h1, h2, p unless you are trying to target only h1's, h2's and p tags with the `.dark-background-text`